.buildkite/pipeline.yml

```diff
@@ -25,5 +25,21 @@ steps:
 
   - label: ":golang-lint: Lint"
     key: lint
-    command: make lint
+    command: rm -rf .git && make lint
 
+  - label: ":github: Release Build"
+    key: "gem-push"
+    if: build.tag != null
+    depends_on:
+     - build
+     - lint
+    command: |
+      docker run -v $(pwd):/app -w /app -e GOOS=darwin -e GOARCH=amd64 golang:1.20 go build -o packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_amd64 -buildvcs=false
+      docker run -v $(pwd):/app -w /app -e GOOS=darwin -e GOARCH=arm64 golang:1.20 go build -o packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_arm64 -buildvcs=false
+
+      zip "packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_amd64.zip" "packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_amd64"
+      zip "packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_arm64.zip" "packer-plugin-hostmgr_${BUILDKITE_TAG}_x5.0_darwin_arm64"
+
+      docker run -v $(pwd):/app -w /app -e BUILDKITE_TAG -e GITHUB_TOKEN ruby:3.2.0 /bin/bash -c "bundle install && bundle exec ruby .buildkite/upload-artifacts.rb"
```

Comment:
Would it make sense to move all of this to the `Makefile`?

Maybe via 2 different targets (one to `go build` + `zip` the 2 architectures, another to `upload-artifacts` (because it could make sense for debugging purposes that we'd want to build from our machines without uploading)

Reply:
Oh nice, that's how I implemented it on #2 😌